external/date.js

```diff
@@ -228,7 +228,7 @@ $.extend( _Date.prototype, {
 	equal: function( other ) {
```

Comment:
My opinion about the `external/date.js` in general?

If it's an external library, it should live in its own repo. If it's an UI internal thing, it should live somewhere in `ui/`.

Just let me know if you were asking about anything else. :)